### PR TITLE
Fix destring

### PR DESCRIPTION
**Pull request by konradzdeb**
Hi, I've fixed a bug in the call to destring as it wasn't evaluating within the data frame environment. I have also fixed a small mistake in `.Rbuildignore`. 

As a wider point, this brings attention to mock tests. Proper tests should catch that but we would have to mock metrics object properly.

### CODE REVIEW

File: R/format_metrics_data.R [hunk 1]
```diff
@@ -55,16 +55,20 @@ format_metrics_range_data <- function(x) {
...
+                lapply(
+                  X = dfs_to_bind,
+                  FUN = function(dfs) {
+                    x <- suppressWarnings(cbind(x_metrics[i,], dfs))
+                    i <<- i + 1
+                    x
+                  }
+                ))
+
+  res <- within(data = res,
```
Comment (konradzdeb) [inline R/format_metrics_data.R]: This is now amended and evaluates within the data object `res`

**Comment by glenn-m** [comment]
Can we add in some extra test cases? 

You can see where you could add in two extra tests [here](https://codecov.io/gh/glenn-m/promR/src/a111df49151b92d9f55f190f4018b3f2af08d87e/R/format_metrics_data.R). We should have a test which passes in a factor at least.

Thanks

**Comment by konradzdeb** [comment]
> Can we add in some extra test cases?

Yes, definitely worth doing. Also the whole testing mechanism could be upgraded to mock API behaviour, just to test the whole pipeline.

**Comment by glenn-m** [comment]
> > Can we add in some extra test cases?
> 
> Yes, definitely worth doing. Also the whole testing mechanism could be upgraded to mock API behaviour, just to test the whole pipeline.

Yeah, I've been trying to think of the best approach to do it. I'll see if I can work out a decent way today, as I'm planning on finishing the metadata query functionality so will need to write tests for it.

**Comment by konradzdeb** [comment]
_Reminder_ problem is in `curl::has_internet` call not being listed as a dependency.

**Comment by konradzdeb** [comment]
I reckon testing merits longer re-work so probably better to complete as one PR (#32).

**Pull request closed without merging**